mail/mail-server/Portfile

```diff
@@ -41,7 +40,7 @@ depends_lib-append      port:dcc \
                         port:sf-pwgen
 
 # these dependencies are from variants that may not be active
-depends_lib-append      port:apache-solr8 \
+depends_lib-append      port:apache-solr9 \
```

Comment:
You are changing a library dependency and thus the `revision` needs to be increased

Reply:
Thanks for pointing this out. I corrected the `dovecot +solr` port variant to use `apache-solr9` and fixed this to a runtime dependency within `mail-server`. All `dovecot` needs to see is solr running on `http://localhost:8983`, so no need to revbump  either `dovecot` or `mail-server`.